recipe/meta.yaml

```diff
@@ -9,10 +9,12 @@ package:
 source:
   fn: scikit-learn-{{ version }}.tar.gz
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: 6f871ab07657becb3cca158dc4af96a1a14c0a360add04664fba325da43537fa
+  sha256: f861b988089b0ccc74c0be10a105ed2fe34a2a608d6f19e67147b4da26e39ac3
+  patches:
+    - disable-0.18.1-failing-tests-on-osx.patch  # [osx]
```

Comment:
Yeah I have a few screw-ups sorry about that, I think it should be fine now but let's see what the CIs have to say about this.